hera_pspec/pspecdata.py

```diff
@@ -1898,16 +1948,20 @@ def pspec(self, bls1, bls2, dsets, pols, n_dlys=None,
             Each tuple should contain a start (inclusive) and stop (exclusive)
             channel used to index the `freq_array` of each dataset. The default
             (None) is to use the entire band provided in each dataset.
-        
+
+        cov_method: string, specify whether covariance matrices should be computed
+        empirically ('empircal') from the data or through propagation of error bars provided by
```

Comment:
Typo 'empircal'. Also, please put the docstring on a new line, so it doesn't break the ReadTheDocs output, e.g.
```py
cov_method : str
    Specify whether...
```

Reply:
This is orphan code for something I've decided to move to a separate pull request. I've removed it.